### PR TITLE
udemy blog

### PR DESCRIPTION
**Pull request by unaso**
@masahirotanaka @argelius please check and approve.

@erisu I mentioned you (once again).

### CODE REVIEW

File: blog/posts/2016-10-14-1.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+![Title](/blog/content/images/2016/Oct/udemymonaca.jpg)
+
+This time, we have a special treat for all beginners who want to develop beautiful apps, but don't know where to start. One of our Senior Software Engineers, Bryan Ellis, created a short Udemy course for those that need a little help getting started with Cordova app development. It takes less than 2 hours and is extremely easy to follow. Without any excuses, it's time to build apps!
+
+<!-- more -->
+
+
+## Follow our short Udemy session and learn how to create Apache Cordova app from scratch!
```
Comment (argelius) [inline blog/posts/2016-10-14-1.markdown]: "apps" or "an Apache Cordova app"

File: blog/posts/2016-10-14-1.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+[Learn to build your Apache Cordova app Udemy course](https://www.udemy.com/learn-to-build-your-first-apache-cordova-app/learn/v4/) is designed for all interested in app development, as well as HTML, CSS, and JavaScript. It is a free course for beginners who need help with the basics.
```
Comment (argelius) [inline blog/posts/2016-10-14-1.markdown]: The formatting here looks broken.

File: blog/posts/2016-10-14-1.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+The couse is available through [Udemy](https://www.udemy.com/), one of most popular online learning and teaching marketplace platforms. Our course is available completely free of charge, therefore don't hesitate to sign up for a free account and access new knowledge without any obligations.
```
Comment (argelius) [inline blog/posts/2016-10-14-1.markdown]: "course"

File: blog/posts/2016-10-14-1.markdown (hunk 2 above)
Comment (argelius) [inline blog/posts/2016-10-14-1.markdown]: Oh, I misunderstood. Looks fine.

**Comment by masahirotanaka** [comment]
Looks good.

**Pull request merged**